ci-operator/config/openshift/backplane-cli/openshift-backplane-cli-main.yaml

```diff
@@ -2,12 +2,12 @@ base_images:
   base:
     name: ubi
     namespace: ocp
-    tag: "8"
+    tag: "9"
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: rhel-8-release-golang-1.21-openshift-4.16
+    tag: rhel-9-release-golang-1.21-openshift-4.16
```

Comment:
Hi @Dee-6777 , given that I don't have much context on this repo, could you help me understand where these tags come from and how they work?

Reply:
The tag refers to a specific version of the ubi image, such as "8" or "9". For instance, tag: 8 refers to the latest available ubi 8 image, while tag: 9 would reference the latest available ubi9.
When changed to 9, CI will pull the corresponding ubi9 image for the build.